cli/chaosli/cmd/create.go

```diff
@@ -410,6 +425,20 @@ func getDiskPressure() *v1beta1.DiskPressureSpec {
 	return spec
 }
 
+func getDiskFailure() *v1beta1.DiskFailureSpec {
+	if !confirmKind("Disk Failure", "Simulates disk failure by applying ebpf disk failure IO to the target") {
+		return nil
+	}
+
+	spec := &v1beta1.DiskFailureSpec{}
+
+	spec.Path = getInput("Add a filter path to target (or leave blank, default=\"/\")",
```

Comment:
should we have / as a default event though it's prevented in the safemod?
But I wouldn't know what path should be selected as a default in this case 😅

Reply:
Good question! 👍 
This description is not valid anymore. The path could not be blank and does not have a default value because it musts to be defined in the disruption spec.